dev/breeze/src/airflow_breeze/utils/confirm.py

```diff
@@ -75,9 +75,8 @@ def user_confirm(
             else:
```

Comment:
Actually I think there is another bug here. 

Default_answer should have two meanings:

1) when timeout is ther and you do not answer, the default one is returned.
2) when you press enter without providing answer and default is set - the default answer should be returned.

Case 2) is not handled here only 1) actually. It would be great to fix it while you are at it.